### DISCUSSION TITLE
Command for going to next workspace search occurrence

### DISCUSSION THREAD

**Discussion by qlp** [helix-editor/helix#9315] [Q&A] [answered]
When I want to do a workspace-level search replace, I want to interactively choose which search hits to replace, and which ones not to replace. I'm trying to achieve this like this:

1. `<space> /`: search for `foo`
2. select the first occurrence
3. `c bar`: replace search hit `foo` with `bar`

After this, I'd like to have a shortcut to go to the next workspace-level search hit, basically like `n` but at the workspace search level. How do I do this?

**Comment by the-mikedavis** [comment, marked as answer]
You can use `<space>'` to re-open the last picker and then `C-n` or tab to select the next result and enter. It isn't currently possible to make a keybinding for this, it would need #1383.

**Reply by qlp** [reply]
Thanks; let's wait for #1383 ⏳

**Comment by buesing** [comment]
For anybody coming from a search engine, this is how to do it with macros:
```
C-p = "@<space>'<C-p><ret>"
C-n = "@<space>'<C-n><ret>"
```
